### PR TITLE
Add cloning instructions to be inside GOPATH

### PR DESCRIPTION
**Pull request by gouthamve**
Fixes: https://github.com/grafana/loki/issues/175

### CODE REVIEW

File: production/README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ Loki can be run in a single host, no-dependencies mode using the following comma
 You need `go` [v1.10+](https://golang.org/dl/) installed locally.
 
 ```bash
+
+$ go get github.com/grafana/loki
```
Comment (vkatsikaros) [inline production/README.md]: In a fresh machine with a fresh user I get with the `got get` command:
```
# added $GOPATH in .bashrc
$ echo $GOPATH 
/home/vangelis/go

$ go version
go version go1.11.4 linux/amd64

# no such dir with a fresh user
$ tree -L 5 ~/go/  
/home/vangelis/go/ [error opening dir]

0 directories, 0 files

$ go get github.com/grafana/loki
package github.com/grafana/loki: no Go files in /home/vangelis/go/src/github.com/grafana/loki

$ tree -L 5 ~/go/
/home/vangelis/go/
└── src
    └── github.com
        └── grafana
            └── loki
                ├── Gopkg.lock
                ├── Gopkg.toml
                ├── LICENSE
                ├── Makefile
                ├── README.md
                ├── cmd
                ├── docs
                ├── loki-build-image
                ├── pkg
                ├── production
                ├── tools
                └── vendor

11 directories, 5 files
```

Comment (tomwilkie) [reply production/README.md]: That looks about right!  The `go get` is there to put the files in the right place, the error can be ignored.

**Comment by tomwilkie** [comment]
Might want to consider `go get`ting the cmd/loki or cmd/promtail, to avoid the error above.  Otherwise LGTM.

**Comment by daixiang0** [comment]
```
$ go get github.com/grafana/loki
can't load package: package github.com/grafana/loki: no Go files in /go/src/github.com/grafana/loki
```

File: production/README.md (hunk 1 above)
Comment (vkatsikaros) [reply production/README.md]: I see, thanks!

**Comment by gouthamve** [comment]
That is not an error but rather Go telling that there are no buildable files in the main directory, it can be ignored.

**Comment by ThoreKr** [comment]
Having non-zero exit codes is rather uncool for scripting.

Would consider this as an error (as bash does as well).

**Pull request merged**